plugins/outputs/application_insights/appinsightsDiagnosticMsg.go

```diff
@@ -0,0 +1,12 @@
+package application_insights
```

Comment:
Rename the source files using snake_case, I would probably call this file `subscriber.go`